data/en/listsome.json

```diff
@@ -5,15 +5,15 @@
 	"member": "list.listSome(closure [, delimiter] [, includeEmptyFields] [, multiCharacterDelimiter] [, parallel] [, maxThreads])",
 	"returns": "boolean",
 	"related": [],
-	"description": "",
+	"description": "This function calls a given closure/function with every element in a given string list and returns true, if one of the closure calls returns true.",
 	"params": [
-		{"name": "list", "description": "", "required": true, "default": "", "type": "string", "values": []},
-		{"name": "closure", "description": "", "required": true, "default": "", "type": "function", "values": []},
-		{"name": "delimiter", "description": "", "required": false, "default": ",", "type": "string", "values": []},
-		{"name": "includeEmptyFields", "description": "", "required": false, "default": "false", "type": "boolean", "values": []},
-		{"name": "multiCharacterDelimiter", "description": "", "required": false, "default": "true", "type": "boolean", "values": []},
-		{"name": "parallel", "description": "", "required": false, "default": "false", "type": "boolean", "values": []},
-		{"name": "maxThreads", "description": "", "required": false, "default": "20", "type": "numeric", "values": []}
+		{"name": "list", "description": "string list to iterate\n\nAlias: string", "required": true, "default": "", "type": "string", "values": []},
+		{"name": "closure", "description": "function/closure that implements the following constructor [function(any value, numeric index,string list, string delimiter):boolean].\n\nAlias: function, callback, udf", "required": true, "default": "", "type": "function", "values": []},
```

Comment:
You can use `callback_params` parameter in param to document the closure's params

Reply:
I don't know how to do that. Do you have an example?

Reply:
Sure (more or less the same as normal params):

```
"callback_params": [
    { "values": [], "default": "", "description": "The value of the first key/value pair", "name": "value1", "type": "any", "required": false },
    { "values": [], "default": "", "description": "The value of the second key/value pair", "name": "value2", "type": "any", "required": false }
]
```